std/range/package.d

```diff
@@ -6269,15 +6269,22 @@ FrontTransversal!(RangeOfRanges, opt) frontTransversal(
 }
 
 // Bugzilla 16442
-@safe unittest
+pure @safe nothrow unittest
 {
     int[][] arr = [[], []];
 
     auto ft1 = frontTransversal!(TransverseOptions.assumeNotJagged)(arr);
     assert(ft1.empty);
 
-    auto ft2 = frontTransversal!(TransverseOptions.enforceNotJagged)(arr);
-    assert(ft2.empty);
+    try
```

Comment:
It's not worth having `nothrow` on the unittest if it makes the example this ugly.